From b6cbdc3d61d2ad7a47b6a27a289d62c1021e97a5 Mon Sep 17 00:00:00 2001
From: Sayan Das <das.sayan128@gmail.com>
Date: Mon, 10 Jan 2022 17:45:55 +0530
Subject: [PATCH 1/2] Modify the file test.txt

---
 test.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test.txt b/test.txt
index 72fe488..708bf62 100644
--- a/test.txt
+++ b/test.txt
@@ -1,2 +1,2 @@
-It is for oonly commit massage
+It is for only commit massage
 Happy new year
-- 
2.25.1

